src/hooks/bridge/useBridgeCallback.ts

```diff
@@ -178,7 +175,15 @@ function useRouterSwap(
                 trade_qty: typedValue,
               },
             })
-            sendTxToKsSetting(chainId, chainIdOut, txHash, from_token, to_token, inputAmountStr, outputAmountStr ?? '')
+            sendTxToKsSetting(
+              chainId,
+              chainIdOut,
+              txHash,
+              from_token,
+              to_token,
+              inputAmountStr,
+              outputInfo?.outputAmount?.toString() ?? '',
```

Comment:
save db raw string

Reply:
Please use `outputAmountStr`.

Reply:
No need save raw string, save formatted string pls.

Reply:
whyyyy.
save raw string, in the later we can format any way we want

Reply:
![image](https://user-images.githubusercontent.com/33005392/200530448-ff181864-155c-488e-9315-9e58605b36c4.png)
I update format here. 
In the future we want to format 10 digit, we don't have to do anything

Reply:
Okay.